apps/workshop/projects/SymbiosisFinance/helpers/utils.ts

```diff
@@ -0,0 +1,29 @@
+import { Context } from '@defiyield/sandbox';
+
+import { ADDRESS, WEEKS_IN_YEAR } from './constants';
+import erc20Abi from '../../../../../packages/abis/erc20.abi.json';
+import veSISDistributorAbi from '../abis/veSISDistributor.json';
+import { BigNumber } from 'ethers';
```

Comment:
Do not import any libs directly, you should use `ethers` from injected `ctx`. Because you will have a build error:
![image](https://user-images.githubusercontent.com/1782530/215445041-5afee409-7165-4f96-ab40-f9998c74e203.png)

Reply:
@witem Please look at question above https://github.com/defiyield-app/defiyield-self-integrations/pull/14#issuecomment-1406854164